docs/assets/snippet.js

```diff
@@ -3,7 +3,99 @@
  * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
  */
 
-/* globals window, document */
+/* globals window, document, setTimeout */
+
+// Show clipboard input notification when user tries to paste a content from MS Word or Google Docs.
+setTimeout( () => {
```

Comment:
We shouldn't execute this code in snippets from https://github.com/ckeditor/ckeditor5-paste-from-office/tree/master/docs/\_snippets/features. I guess the easiest way to prevent showing the notification on those demos is to register something like `window.preventPasteFromOfficeNotification = true` in those 2 snippets and abort in this `setTimeout()` when this variable is `true`.

Reply:
Done here https://github.com/ckeditor/ckeditor5-paste-from-office/pull/95